### ISSUE TITLE
Grammar validation(Pseudo classes) support

### ISSUE THREAD

**Issue by stefango** [validator/validator#1159]
https://github.com/less/less.js/issues/3620
https://validator.w3.org/nu/#textarea
```css
// wrong（no error reported in this validator）
:not(first-child) {
  background: red;
}

// correct
:not(:first-child) {
  background: red;
}
```

**Comment by sideshowbarker** [comment]
Please re-raise this issue against the CSS validator at https://github.com/w3c/css-validator/issues/new.

(For CSS checking, the HTML checker relies uses the CSS validator code as a library — so after any change for this gets made to the upstream CSS-validator code, we will just pull that change into the HTML checker.)
